jwql/website/apps/jwql/views.py

```diff
@@ -594,7 +594,7 @@ def instrument(request, inst):
     return render(request, template, context)
 
 
-def jwqldb_table_viewer(request):
+def jwqldb_table_viewer(request, tablename_param=None):
```

Comment:
Are there changes that actually call this function with the second parameter?

Reply:
Yes, when you are coming from the monitor view, the tablename is passed as a url parameter which is passed to this view. But if you are accessing the JWQLDB view from the navbar/other means that aren't directly linked from a monitoring page, the default is set to `None`

Reply:
Great thanks!